tests/test_bug_creation.py

```
@@ -0,0 +1,424 @@
+        )
+        with patch("jenkins_job_insight.bug_creation.httpx.AsyncClient") as MockClient:
+            mock_client = AsyncMock()
+            mock_client.get.return_value = mock_response
+            mock_client.__aenter__ = AsyncMock(return_value=mock_client)
+            mock_client.__aexit__ = AsyncMock(return_value=False)
+            MockClient.return_value = mock_client
+
+            results = await search_github_duplicates(
+                title="Login fails with valid credentials",
+                repo_url="https://github.com/org/repo",
+                github_token="ghp_test",
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to argument: "github_token"

(S106)